static/docs/user-guide/contributing-documentation.md

```diff
@@ -0,0 +1,113 @@
+# Contributing Documentation
+
+We welcome contributions to [DVC.org](https://github.com/iterative/dvc.org) by
+the community.
+
+## How to report a problem
+
+Please search [issue tracker](https://github.com/iterative/dvc.org/issues)
+before creating a new issue. This issue can be about a typographical error, an
+updation of the documentation or simply an improvement in generation of
+documentation.
+
+If you feel that you can fix or implement it, please read a few paragraphs below
+to learn how to submit your changes.
+
+## Structure of project
+
+* [Content](https://github.com/iterative/dvc.org/tree/master/static/docs) -
```

Comment:
Are this directories? It is not that clear.

Reply:
@mroutis Somewhat, these three bullet points explain the 3 parts of the project. The docs (static markdown files), media files and sections (the navigation menu).